### PR TITLE
#3775 - Phenylalanine mustard is displayed incorrectly in the Structure library

### PR DESCRIPTION
**Pull request by baranovdv**
- fix library.sdf phenylalanine mustard description

## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [ ] e2e-tests written
- [ ] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [ ] task status changed to "Code review"
- [ ] reviewers are notified about the pull request

### CODE REVIEW

File: ketcher-autotests/tests/Templates/Template-Library/template-library.spec.ts-snapshots/Templates---Template-Library-Template-with-chiral-flag-0-with-ignoreChiralFlag-enabled-disabled-1-chromium-linux.png
Comment (rrodionov91) [inline ketcher-autotests/tests/Templates/Template-Library/template-library.spec.ts-snapshots/Templates---Template-Library-Template-with-chiral-flag-0-with-ignoreChiralFlag-enabled-disabled-1-chromium-linux.png:1]: I believe it should look like this
image

Please clarify the correctness of structure

**Review by rrodionov91 — CHANGES_REQUESTED** [review]

File: ketcher-autotests/tests/Templates/Template-Library/template-library.spec.ts-snapshots/Templates---Template-Library-Template-with-chiral-flag-0-with-ignoreChiralFlag-enabled-disabled-1-chromium-linux.png
Comment (baranovdv) [reply ketcher-autotests/tests/Templates/Template-Library/template-library.spec.ts-snapshots/Templates---Template-Library-Template-with-chiral-flag-0-with-ignoreChiralFlag-enabled-disabled-1-chromium-linux.png:1]: Updated structure:

Снимок экрана 2024-03-22 в 01 37 45

from here:
https://pubchem.ncbi.nlm.nih.gov/substance/374913453

Looks better

**Review by NataliaLoginova — APPROVED** [review]

File: ketcher-autotests/tests/Templates/Template-Library/template-library.spec.ts-snapshots/Templates---Template-Library-Template-with-chiral-flag-0-with-ignoreChiralFlag-enabled-disabled-1-chromium-linux.png
Comment (baranovdv) [reply ketcher-autotests/tests/Templates/Template-Library/template-library.spec.ts-snapshots/Templates---Template-Library-Template-with-chiral-flag-0-with-ignoreChiralFlag-enabled-disabled-1-chromium-linux.png:1]: updated to such structure:
Снимок экрана 2024-03-22 в 17 54 39

according to Olga Nazarenko recommendation

Comment (baranovdv) [reply ketcher-autotests/tests/Templates/Template-Library/template-library.spec.ts-snapshots/Templates---Template-Library-Template-with-chiral-flag-0-with-ignoreChiralFlag-enabled-disabled-1-chromium-linux.png:1]: new update from Olga Nazarenko:
Снимок экрана 2024-03-22 в 18 37 44

**Review by rrodionov91 — APPROVED** [review]

**Pull request merged**